### PR TITLE
MagicAPI

### PR DESCRIPTION
**Pull request by Alphabetsoup16**

### CODE REVIEW

**Comment by 444B** [comment]
@Alphabetsoup16 Hey man
Can you summarize briefly the changes made? I can see that main.py has been commented out and we are now using app.py for the main function, correct?
By the way, I think the merge reviews are great because it will ensure that we are all on the same page as the project grows in complexity

**Review by 444B — COMMENTED** [review]
Yeah im gonna need a breakdown of app.py
Testing making a comment from the review section

**Comment by Alphabetsoup16** [comment]
> @Alphabetsoup16 Hey man Can you summarize briefly the changes made? I can see that main.py has been commented out and we are now using app.py for the main function, correct? By the way, I think the merge reviews are great because it will ensure that we are all on the same page as the project grows in complexity

I thought it would be good going forward as well! So instead of using SQLfunctions, now instead we are going to create an API that can handle requests/responses. You are correct that everything is happening in the app.py and I left the other files for reference commented out but will delete them before merging this branch. I updated the main branch readme to include instructions on running the API locally. Let me know if you have anymore questions. I'd love some help brainstorming more API methods to add.

**Review by 444B — COMMENTED** [review]
Thanks! Please add some comment inside the code so we can contextualize the function

**Review by 444B — APPROVED** [review]
